react/OrderBy.js

```diff
@@ -9,6 +9,10 @@ export const SORT_OPTIONS = [
     value: '',
```

Comment:
Maybe a better solution would be to set this value as `OrderByScoreDESC`. Duplicating it is not an option. @victorhmp can you give some input on this?

Reply:
Sure!
I think the best solution would simply be to not change anything here in `SORT_OPTIONS` and add a "translation" step in the component itself. Something along the lines of

```ts
if (receivedOrderBy === 'OrderByScoreDES') {
  orderBy = ''
}
```

The thing is, this option of `OrderByScoreDESC` used to be our default. But this is also how the Catalog API returns products, except when showing products inside a collection since collections can have custom product orderings. Basically, the ordering people see in search result pages was not affect at all, unless they're seeing a collection's page, but in that case, it changed for the better.

We need to keep this `value: ''` so that we send an empty string as the value for the `orderBy` query variable. This way the API just returns products ordered according to its default, exactly as we want.

Reply:
Awesome, thanks! @italo-batista, can you do that?

Reply:
Of course, I'll send a new PR with this.